tools/flaky_test_monitor/testdata/summary2/test3-multi-no-result-tests/expected-output/test3-multi-no-result-tests.json

```diff
@@ -1,229 +0,0 @@
-{
```

Comment:
was this deleted file replaced? I can see you changed the names of the `no_result` field to `exception` but I don't see a replacement for this file